packages/solidity-contracts/protocol/utils.ts

```diff
@@ -1,6 +1,9 @@
 // Helper functions for testing
-import { BigNumber as BN } from 'ethers';
+import type { BigNumberish, BytesLike } from 'ethers';
+import { BigNumber as BN, BigNumber, ethers } from 'ethers';
```

Comment:
```suggestion
import type { BigNumberish, BytesLike } from 'ethers';
import { BigNumber as BN, ethers } from 'ethers';
```

Haven't seen `import type` before. Can these two imports be merged?

Reply:
To my knowledge they can be merged, but ESLint will complain about it. It was a rule introduced by @luizstacio if I recall correctly, it might have something to do with how the monorepo packages the typings etc.